charts/lnd/templates/lnd-pass-secret.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ printf "%s-pass" (include "lnd.fullname" .) }}
+  labels:
+    {{- include "lnd.labels" . | nindent 4 }}
+stringData:
+  password: {{ default "randAlphaNum 6" .Values.walletPassword | quote }}
```

Comment:
there should be a lot more characters because 6 can easily be brute force. 24 at least.